google-cloud-storage/src/main/java/com/google/cloud/storage/StorageException.java

```diff
@@ -43,7 +46,9 @@
           new Error(500, null),
```

Comment:
Where does this Error class come from?

Reply:
https://github.com/googleapis/java-core/blob/master/google-cloud-core/src/main/java/com/google/cloud/BaseServiceException.java#L174

Reply:
Also that's marked `@InternalApi` so it shouldn't be used here at all.

Reply:
StorageException is also annotated with `@InternalApi`. I'm not introducing the use Error(). I think this is okay, as Storage isn't the only library using the base class `BaseHttpServiceException`, e.g. [Bigquery](https://github.com/googleapis/java-bigquery/blob/master/google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/BigQueryException.java#L34).

Reply:
For `@InternalApi` from google-cloud-core, it's public because it needs to be, but intended for use by our libraries

Reply:
I'll live with it this PR, but that's a major abuse of `@InternalApi`. It's as effectively locked in as any other public API. We should admit what's been done here, and remove `@InternalApi` from those classes. We can't change them absent an incompatible major version bump. :-(